### PR TITLE
Moved splashscreen css to css file

### PR DESCRIPTION
**Pull request by ckhatri**
Splashscreen css is currently in the html file, to clean up code decided to move it to the app.css file. 

I also moved splashscreen.png to the images folder.

### CODE REVIEW

File: client/app/assets/app.css [hunk 1]
```diff
@@ -6,6 +6,8 @@ body {
     margin: 0px;
     padding: 0px;
     background-position: center;
+    background-color: #000;
+    border-radius: 10px;
```
Comment (spresnal) [inline client/app/assets/app.css]: The border-radius wasn't being applied to the body before. Was this an intended change?

Comment (ckhatri) [reply client/app/assets/app.css]: nice catch, fixed!

**Comment by luciorubeens** [comment]
can you fix the conflict?

**Comment by ckhatri** [comment]
Should be fixed now!

**Comment by luciorubeens** [comment]
Hmm I tried it and we can't do this because the css is loaded before the splash screen..

**Pull request closed without merging**